packages/lesswrong/lib/betas.ts

```diff
@@ -22,7 +22,7 @@ const isEAForum = forumTypeSetting.get() === 'EAForum'
 // Features in progress                                                     //
 //////////////////////////////////////////////////////////////////////////////
 
-export const userHasCommentOnSelection = isEAForum ? disabled : shippedFeature;
+export const userHasCommentOnSelection = isEAForum ? optInOnly : shippedFeature;
```

Comment:
I woulda sworn this was optInOnly on the Forum already. TBH, at this point I would ship it. The feature is ~complete on LW, and seems net positive. There's not a lot of value in putting things in optInOnly unless you know what you're looking for, IMO.

Reply:
Viewing side comments was opt-in, but making them was disabled... which overall meant they were just disabled

Reply:
Not quite! Because of block quotes. But now I understand.